includes/class-wc-payments-checkout.php

```diff
@@ -299,7 +299,7 @@ public function get_payment_fields_js_config() {
 	 */
 	public function get_enabled_payment_method_config() {
 		$settings                = [];
-		$enabled_payment_methods = $this->gateway->get_upe_enabled_payment_method_ids_based_on_manual_capture();
+		$enabled_payment_methods = $this->gateway->get_payment_method_ids_enabled_at_checkout();
```

Comment:
Before: only checking based on manual capture.
Now: the `get_payment_method_ids_enabled_at_checkout` method _also_ checks based on currency, other than manual capture eligibility.